format.html

```diff
@@ -0,0 +1,83 @@
+<html>
+
+<head>
+
+</head>
+
+<body>
+    <div>
+        <ul>
+            <li>Sunday</li>
+            <li>Monday</li>
+            <li>Tuesday</li>
```

Comment:
second activity is not done . add html comment before each list
`<!-- Write your comments here -->`